core/templates/dev/head/css/oppia.css

```diff
@@ -4361,7 +4390,7 @@ md-card.preview-conversation-skin-supplemental-card {
 }
 
 @media only screen and (max-width: 768px) {
-  .oppia-about-content, .oppia-about-extra-content {
+  .oppia-about-content, .oppia-static-extra-content {
```

Comment:
Ditto. Please make sure you grep for other things you've changed and that there are no more residual instances.